Gruntfile.js

```diff
@@ -120,7 +120,8 @@ module.exports = function(grunt) {
                     src: ["app/built.js",
                         "app/data/**",
                         "img/**",
-	                    "audio/**",
+                        "video/**",
```

Comment:
If these videos are now being copied to www/, then the lower command which copies them to /res/raw should be removed.